src/main/java/sorald/processor/ThreadLocalWithInitial.java

```diff
@@ -0,0 +1,71 @@
+package sorald.processor;
+
+import java.util.List;
+import java.util.function.Supplier;
+import sorald.annotations.ProcessorAnnotation;
+import spoon.reflect.code.CtExpression;
+import spoon.reflect.code.CtInvocation;
+import spoon.reflect.code.CtLambda;
+import spoon.reflect.code.CtNewClass;
+import spoon.reflect.code.CtReturn;
+import spoon.reflect.code.CtStatement;
+import spoon.reflect.code.CtTypeAccess;
+import spoon.reflect.declaration.CtClass;
+import spoon.reflect.reference.CtExecutableReference;
+import spoon.reflect.reference.CtTypeReference;
+
+@ProcessorAnnotation(key = "S4065", description = "\"ThreadLocal.withInitial\" should be preferred")
+public class ThreadLocalWithInitial extends SoraldAbstractProcessor<CtNewClass<?>> {
+
+    @Override
+    protected void repairInternal(CtNewClass<?> newClass) {
+        CtClass<?> innerClass = newClass.getAnonymousClass();
+        CtExecutableReference<?> initialValueMethod = findInitialValueMethod(innerClass);
+        CtLambda<?> lambda = createSupplier(initialValueMethod);
+        CtInvocation<?> invocation = createinitialMethod(newClass, lambda);
+        invocation.setArguments(List.of(lambda));
+        newClass.replace(invocation);
+    }
+
+    private CtInvocation<?> createinitialMethod(CtNewClass<?> threadLocal, CtLambda<?> lambda) {
```

Comment:
Shouldn't be `createInitialMethod`? (typo on first character of `initial`)

Reply:
@MartinWitt I think I overlooked this while suggesting the typographical error.